dashboard/db/migrate/20200722182927_create_unit_groups.rb

```diff
@@ -0,0 +1,27 @@
+class CreateUnitGroups < ActiveRecord::Migration[5.0]
+  def up
+    # create unit_groups as an exact copy of the courses table
+    execute 'CREATE TABLE unit_groups LIKE courses'
+    execute 'INSERT INTO unit_groups SELECT * FROM courses'
```

Comment:
This strategy was borrowed from https://stackoverflow.com/questions/3280006/duplicating-a-mysql-table-indices-and-data/3280042#3280042